timeline.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title>Timeline</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="portfoliostyle.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	
+</head>
+<body>
+	<!-- navbar -->
+	<nav>
+		<a href="portfolio.html" class="nav-item">Home</a>
+		<a href="eleanor-roberts-cv.pdf" class="nav-item cv" target="_blank">CV</a>
+	</nav>
+	<!-- header -->
+	<header id="home">
+		<h1>History of Me</h1>
+	</header>
+	<!-- timeline -->
+	<section id="timeline">
```

Comment:
Do you need this section and/or the first div child? Could you combine the style properties :)

Reply:
Ahh woops see I've commented on this before so you must need it :)

Reply:
Actually I think I missed it! I have now removed the extra unnecessary div 😅